webui/api-ts-sdk/Makefile

```diff
@@ -1,14 +1,16 @@
 PROJECT_ROOT := "../.."
+CODEGEN_BIN := swagger-codegen-cli-2.4.14.jar
 
 .PHONY: all
 all: get-deps build
 
+deps/${CODEGEN_BIN}:
+	mkdir -p deps
+	curl https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.14/${CODEGEN_BIN} \
```

Comment:
we could make version a var too :thinking:

Reply:
I'd be mildly in favor of that, but it doesn't matter much.

Reply:
its benefit I think would depend on whether they continue to host the jar file with the same format or not. I'll skip it